README.md

```diff
@@ -88,55 +88,6 @@ jobs:
 ```
   * [GITHUB_WORKSPACE](https://help.github.com/en/github/automating-your-workflow-with-github-actions/virtual-environments-for-github-hosted-runners) is the environment variable provided by GitHub which represents the root of your repository.
 
-### Configure Azure credentials as GitHub Secret:
-
-To use any credentials like Azure Service Principal,add them as [secrets](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets) in the GitHub repository and then use them in the workflow.
-
-Follow the steps to configure the secret:
-  * Define a new secret under your repository settings, Add secret menu
-  * Store the output of the below [az cli](https://docs.microsoft.com/en-us/cli/azure/?view=azure-cli-latest) command as the value of secret variable 'AZURE_CREDENTIALS'
-```bash
-
-   az ad sp create-for-rbac --name "myApp" --role contributor \
-                            --scopes /subscriptions/{subscription-id}/resourceGroups/{resource-group} \
-                            --sdk-auth
-
-  # Replace {subscription-id}, {resource-group} with the subscription, resource group details
-
-  # The command should output a JSON object similar to this:
-
-  {
-    "clientId": "<GUID>",
-    "clientSecret": "<GUID>",
-    "subscriptionId": "<GUID>",
-    "tenantId": "<GUID>",
-    (...)
-  }
-
-```
-  * Now in the workflow file in your branch: `.github/workflows/workflow.yml` replace the secret in Azure login action with your secret (Refer to the example above)
-
-## Azure CLI Action metadata file
-
-```
-# File: action.yml
-
-# Automate your GitHub workflows using Azure CLI scripts.
-name: 'Azure CLI'
-description: 'The action is used to execute Azure CLI commands'
-inputs:
-  inlineScript:
-    description: 'Specify the script here'
-    required: true
-  azcliversion:
-    description: 'Azure CLI version to be used to execute the script'
-    required: false
-    default: 'latest'
-runs:
-  using: 'node16'
-  main: 'lib/main.js'
-```

```

Comment:
Just out of curiosity, what is the usage of this **metadata** section?

Reply:
The **metadata** section is redundant, since the metadata information can be found in `action.yml`. If it is reserved, we need to update `README.md` each time `action.yml` is changed.